lstchain/reco/dl1_to_dl2.py

```diff
@@ -402,6 +402,16 @@ def build_models(filegammas, fileprotons,
         lhfit_df_proton = pd.read_hdf(fileprotons, key=dl1_likelihood_params_lstcam_key)
         df_proton = pd.concat([df_proton, lhfit_df_proton], axis=1)
 
+    # Normalize all azimuth angles to the range [0, 360) degrees
```

Comment:
This transformation should not be applied when training model, but in the DL1 parameters construction.
The risk is to have different parameters at training and inference.

Reply:
Alternatively, one could have a `preprocess` step, including filtering and this type of transformation (including also lines 409-413), that should be applied at training and inference.

Reply:
Here I simply moved this block of code, but I agree it would be better in an earlier step.
Changing it now may lead to breaking changes. Should it be opened as an issue for the next major release?

Reply:
Indeed I did not realise you just moved it.
Yes please open an issue, I change my review.